change/@ni-nimble-components-be528419-2f60-4d4b-b6f0-a8b8e1e79bae.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
It'd be good to make sure the react wrappers aren't ending up in the built `nimble-components` package. You could do that by manual inspection of the built package (it should be available as a pipeline artifact). I'm not sure what we could do to make sure they aren't accidentally included later. Maybe better alarming when the package size increases?

Unless someone else has ideas I'm ok with one time manual verification for now. Longer term is the idea they'd go in a completely separate package? But that's too much work because it leads to a circular dependency? (storybook in nimble-components) -> (react components) -> (components in nimble components).

Reply:
I've confirmed that the react wrappers aren't ending up in the built `nimble-components` package in the current setup. The rollup config is targeting the components `index.ts` file, so I don't expect the react wrapper to easily get sucked into the package. 🤷‍♂️

I tried to autogenerate the wrappers, but failed to make much progress by extending the rollup config. I too, am OK with the manual approach for now… it's literally 10 seconds of work for a new component, and shouldn't change much.